server/test/app/SecurityBrowserTest.java

```diff
@@ -32,9 +32,13 @@ public void getApplicantRepository() {
     userRepository = app.injector().instanceOf(UserRepository.class);
   }
 
-  protected void loginWithSimulatedIdcs() {
+  private void loginWithSimulatedIdcs() {
     goTo(routes.LoginController.applicantLogin(Optional.empty()));
     // If we are not cookied, enter a username and password.
+    // Otherwise, since the fake provider uses the "web" flow, we're automatically sent to the
+    // redirect URI to merge logins.
+    // TODO(#1770): Once the user is logged out from the fake OIDC provider, the below conditional
```

Comment:
Note to the reviewer: Once #3171 adds support for logging out from the central provider, this code may be able to be simplified.

Reply:
And hopefully any breakages as a result would be caught due to the changes to build the OIDC container as part of running unit tests.